include/openthread/ip6.h

```diff
@@ -906,6 +906,8 @@ typedef struct otBorderRoutingCounters
     otPacketsAndBytes mInboundMulticast;  ///< The counters for inbound multicast.
     otPacketsAndBytes mOutboundUnicast;   ///< The counters for outbound unicast.
     otPacketsAndBytes mOutboundMulticast; ///< The counters for outbound multicast.
+    otPacketsAndBytes mInboundInternet;   ///< The counter for inbound Internet when DHCPv6 PD enabled.
```

Comment:
How does this work along with other counters? Is an inbound unicast message counted in `mInboundUnicast` and `mInboundInternet`? 

How do we tell msg is from internet? May be good to clarify this?

Reply:
The key question for me is how do we want to define `mInboundInternet` and `mOutboundInternet`? What traffic/message are we including in this definition?

For example, a Thread device (after getting DHCP6 PD address) can send message to any server on internet (say a Google device sends info to a Google server). This would be technically outbound traffic to internet, but I don't think the address `/48` bit prefix match would work for this case.

Reply:
I'm monitoring the in/out traffic on wpan0 in this case. So when it's inbound traffic, the source address must an Internet address, and when it's outbound the destination should be an Internet address. Can you give me more details about your instructions? eg. the corner cases I can't cover, or the current assertion calculated more counters than I expected?